plutus-wallet-api/ledger/Ledger/Orphans.hs

```diff
@@ -19,13 +19,23 @@ import qualified Language.PlutusTx.Prelude  as P
 import           Schema                     (FormSchema (FormSchemaHex), ToSchema (toSchema))
 import           Type.Reflection            (Typeable)
 
+
+{- Serialising Digests from Crypto.Hash.  This is more complicated than
```

Comment:
Make this a proper note i.e. begin with `{- Note [Serialising Digetss from Crypto.Hash` on it's own line. Makes searching for them much easier!